config/software/openssl-customization.rb

```diff
@@ -60,7 +60,7 @@ def get_sanitized_rbconfig(config)
       # to pick up our script which find the CA bundle in omnibus installations and points SSL_CERT_FILE to it
       # if it's not already set
       source_openssl_rb = File.join(embedded_ruby_lib_dir, "openssl.rb")
-      File.open(source_openssl_rb, "a") do |f|
+      File.open(source_openssl_rb, "r+") do |f|
```

Comment:
I think this is gonna overwrite this many characters at the beginning of the file, which will be weird. It'll probably be functional, but only because there's enough bytes of license/author info at the top. Might be cleaner to do:

```ruby
existing_content = f.read
f.rewind
f.write("\nrequire 'ssl_env_hack'\n")
f.write(existing_content)
```